src/commands/cereal/drink.ts

```diff
@@ -84,56 +84,59 @@ const parseObjectToDrink = (drink_res: Object): Drink => {
   const category = drink_res["strCategory"];
   const instructions = drink_res["strInstructions"];
   const thumb_nail = drink_res["strDrinkThumb"];
-  const glas = drink_res["strGlass"];
+  const glass = drink_res["strGlass"];
   const picture = drink_res["strImageSource"];
 
   const ingredients = parseIngredientsFieldsList(drink_res);
 
-  const drink = new Drink(_.toInteger(id), name, category, instructions, thumb_nail, glas, ingredients, picture);
+  const drink = new Drink(_.toInteger(id), name, category, instructions, thumb_nail, glass, ingredients, picture);
   return drink;
 };
 
-const createDrinkEmbed = async (result: Drink): Promise<{ embeds: MessageEmbed[] }> => {
+const createDrinkEmbed = async (result: Drink): Promise<MessageEmbed> => {
```

Comment:
Yes, that is correct. I moved the creation of the object to the `dispatch` function because I wanted to create multiple drinks together with one dispatch call. So it is just handier that way. The result is still the same.

Reply:
Ah yeah, I missed that. Makes sense now.